NuRadioReco/framework/base_trace.py

```diff
@@ -129,7 +129,7 @@ def get_hilbert_envelope(self):
         return np.abs(h)
 
     def get_hilbert_envelope_mag(self):
-        return np.linalg.norm(self.get_hilbert_envelope(), axis=0)
+        return np.linalg.norm(self.get_hilbert_envelope(), axis=-1)
```

Comment:
I still don't think this is right. For a 3D trace, this will now take the norm over the trace length and return a (3)-shaped array, for a 1D array it will return a scalar. I still think it shouldn't do anything in 1D, and take the norm over the 3 dimensions for a 3D trace...

Reply:
Ahh... should have paid more attention to what I am doing. You are completely right :)